### PR TITLE
footer tweaks, added title and ellipsis to url and namespace

### PR DESCRIPTION
**Pull request by jcavalieri**

### CODE REVIEW

**Comment by jcavalieri** [comment]
screen shot 2015-11-06 at 3 24 34 pm

File: lib/sidekiq/web_helpers.rb [hunk 1]
```diff
@@ -245,5 +245,10 @@ def environment_title_prefix
     def product_version
       "Sidekiq v#{Sidekiq::VERSION}"
     end
+
+    def redis_connection_and_namespace
+      namespace_suffix = namespace.blank? ? '' : "##{namespace}"
+      "#{redis_connection}#{namespace_suffix}"
```
Comment (mperham) [inline lib/sidekiq/web_helpers.rb]: Could you memoize this so its not constructed twice?

Comment (jcavalieri) [inline lib/sidekiq/web_helpers.rb]: Sure. I'm away from my desk right now but will do.

_______________
John Cavalieri

**Comment by jcavalieri** [comment]
not sure why the CI build seems to taking a while

@mperham, I memoized the string as requested

**Pull request merged**